jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -179,6 +183,60 @@ protected ScopedContext newContext()
         return new ScopedContext();
     }
 
+    @ManagedAttribute(value = "temporary directory location", readonly = true)
+    public File getTempDirectory()
```

Comment:
So this can be null? As sometimes methods called getTempDirectory() never return null, it would be good to javadoc when they do and when they don't.